### ISSUE TITLE
undefined reference to `strsep'

### ISSUE THREAD

**Issue by Mihaylov93** [SimulIDE/SimulIDE#20]
```
g++ -Wl,-s -Wl,-subsystem,windows -mthreads -o release\simulide.exe @object_script.simulide.Release  -lwsock32 D:\msys64\mingw32\lib\libQt5Svg.dll.a D:\msys64\mingw32\lib\libQt5Widgets.dll.a D:\msys64\mingw32\lib\libQt5Multimedia.dll.a D:\msys64\mingw32\lib\libQt5Gui.dll.a D:\msys64\mingw32\lib\libQt5Xml.dll.a D:\msys64\mingw32\lib\libQt5Script.dll.a D:\msys64\mingw32\lib\libQt5Concurrent.dll.a D:\msys64\mingw32\lib\libQt5SerialPort.dll.a D:\msys64\mingw32\lib\libQt5Network.dll.a D:\msys64\mingw32\lib\libQt5Core.dll.a  -lelf -lglu32 -lopengl32 -lgdi32 -luser32 -lmingw32 D:\msys64\mingw32\lib\libqtmain.a -lshell32 
D:/msys64/mingw32/bin/../lib/gcc/i686-w64-mingw32/10.2.0/../../../../i686-w64-mingw32/bin/ld.exe: release/sim_utils.o:sim_utils.c:(.text+0x8c): undefined reference to `strsep'
collect2.exe: error: ld returned 1 exit status
mingw32-make[1]: *** [Makefile.Release:1410: release/simulide.exe] Error 1
```

**Comment by pratikpc** [comment]
Seems to be an issue with sim_avr

**Comment by Mihaylov93** [comment]
No, its a windows issue by not implementing strsep in strings.h

**Comment by pratikpc** [comment]
@Mihaylov93 yes but the only file that uses it is at 

1. https://github.com/SimulIDE/SimulIDE/blob/de80a184ac07bbce80f26ab083b5f24c0bfba1f3/src/simavr/sim/sim_utils.c#L63

In the SimAVR folder. I believe the development of simavr is outside the scope of this fork.

So we need to check if the actual changes have been applied at the SimAVR parent repo

**Comment by Mihaylov93** [comment]
I already have a working (Compiling and running) windows compilation with both #21  and this issue solved. Just opened this issues for documentation purposes.

**Comment by Mihaylov93** [comment]
Solved by #25
